### PR TITLE
Upgrade to Ruby 2.6.2

### PR DESCRIPTION
**Pull request by jturkel**
I'm going with Ruby 2.6.1 for now. Ruby 2.6.2 has been released recently but a corresponding Docker image hasn't been released yet.

@atsheehan - you're prime

### CODE REVIEW

**Comment by jturkel** [comment]
Looks like a rubocop upgrade is required for this. I'll spin up a separate PR to land that first.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # To build run: docker build -t avro-schema-registry .
 
-FROM ruby:2.4.2
+FROM ruby:2.6.1
```
Comment (tjwp) [inline Dockerfile]: If there is no ruby:2.6.2 image yet, you might want to add a `RUN gem update --system` line to address the Rubygems vulnerabilities: https://blog.rubygems.org/2019/03/05/security-advisories-2019-03.html

**Comment by jturkel** [comment]
See #133 for the rubocop upgrade

File: Dockerfile (hunk 1 above)
Comment (jturkel) [reply Dockerfile]: According to https://github.com/docker-library/ruby/issues/273 it was already handled in a newer version of the `ruby:2.6.1` image.

Comment (jturkel) [reply Dockerfile]: Looks like the 2.6.2 Docker image has been published so we can just go to that :)

**Comment by jturkel** [comment]
@atsheehan - I moved to Ruby 2.6.2 since the Docker image was released. Still look good to you?

**Pull request merged**